tensorboard/plugins/projector/vz_projector/vz-projector-inspector-panel.ts

```diff
@@ -282,6 +365,10 @@ namespace vz_projector {
       this.searchBox.message = '';
       const minDist = neighbors.length > 0 ? neighbors[0].dist : 0;
 
+      if (this.spriteImagesAvailable && this.showNeighborImages) {
+        var image_renderer = this.spriteImageRenderer();
```

Comment:
Okay—this is still hoisted, but it’s just one variable and much clearer,
so I’ll allow it. :+1:

Reply:
hahaha, yeah I'm not sure how to avoid that, other than putting the conditional inside the renderer, but that would make things a lot less clear so,,

Reply:
Yep, no problem.